doc/installation.rst

```diff
@@ -34,25 +38,35 @@ Just visit the `Spyder page on Binder`_ to get started using Spyder now.
 
 
 
-.. _standalone_installers_ref:
+.. _install-standalone:
 
 =====================
 Standalone installers
 =====================
 
-Our standalone installers for Windows and macOS are available from Spyder 4.2 onwards.
-We recommend using this installation method on those platforms, but we offer several other options for Linux, advanced users and specific needs, so keep reading if that's the case for you.
+The standalone installers are our recommended method for most users on Windows and macOS, with experimental Linux support under active development.
+They work much like installing any other IDE, where Spyder can be installed and updated independently of the Python environments you use to run your code, avoiding problems with incompatible packages and broken installations that users often face when doing so within their own Python installation.
```

Comment:
```suggestion
They work much like installing any other IDE, where Spyder can be installed and updated independently of the Python environments you use to run your code, avoiding problems with incompatible packages and broken installations that users often face when doing so within Anaconda, for instance.
```

* Avoids repeating `installation` twice.
* I think it provides a simpler reference (i.e. Anaconda) for users to understand what we mean here.

Reply:
Okay, though its a little weird to say "for instance" here. I've reworded it accordingly, and also split it into two shorter sentences and avoided another repeat of "installing".

```suggestion
They work much like for any other IDE, where Spyder can be installed and updated independently of the Python environments you use to run your code.
This avoids the problems with incompatible packages and broken installations users often face when mixing Spyder with the (Conda, etc) environments they use to run their code.
```